src/algorithms/graph-builder.ts

```diff
@@ -23,20 +23,24 @@ import Graph from './models/graph';
 export default (nodes: INode[], ways: IWay[]): Graph => {
   const vertices = nodesToVertices(nodes);
   const graph = new Graph();
+
   graph.addVertex(...vertices);
+
   for (const way of ways) {
     const oneway = isOneWay(way);
     const nodeIds = way.loc.nodes;
+
     for (let i = 0; i < nodeIds.length - 1; i++) {
       const cost = new EdgeCost();
       const v1 = graph.getVertex(nodeIds[i]);
       const v2 = graph.getVertex(nodeIds[i + 1]);
+
       cost.distance = getDistance(v1, v2);
-      if (!oneway) {
-        v2.addNeighbor(cost, v1);
-      }
+
+      if (!oneway) { v2.addNeighbor(cost, v1); }

```

Comment:
Not a fan of this :-/

Reply:
I'm not against one or another. In this case, I was just trying to simplify the whole function, but since it seems like this form is not used elsewhere I'll probably revert it   😉